PORTAL/jobs/_pyperformance.py

```diff
@@ -3245,6 +3248,8 @@ def from_remote(  # type: ignore[override]
     ) -> "PyperfResultsRepo":
         if not remote:
             remote = cls.REMOTE
+        if root is None:
+            root = cls._DEFAULT_ROOT
```

Comment:
It may make more sense to look to the config file for the default before falling back to the current behavior (a hard-coded default).  So consider adding a new config option for this instead of the monkey-patching approach.

I guess the default from the config would be passed in rather than the config getting used directly here.

Reply:
Yeah, that probably makes sense.  The tests would just set it to another subdirectory in the tree.  Will require some refactoring to pass the cfg here.

Reply:
looks good